gui/src/components/s3/account-management.vue

```diff
@@ -468,6 +477,122 @@
         </cortx-has-access>
       </v-col>
     </v-row>
+    
+    <v-dialog 
+      v-model="showResetAccountForm" 
```

Comment:
Name "showResetAccountForm" is more looks like a function and not a model.

Reply:
Updated name 'showResetPasswordDialog'